operator/hack/common.sh

```diff
@@ -34,26 +35,27 @@ function apply_operator_manifests() {
   catalog_source_crd="$("${ROOT_DIR}/operator/hack/retry-kubectl.sh" < /dev/null get customresourcedefinitions.apiextensions.k8s.io catalogsources.operators.coreos.com -o yaml)"
   local disable_security_context_config="# "
   local has_scc_key
+  local operator_channel
+  local install_plan_approval
+  local starting_csv
```

Comment:
Please move `install_plan_approval` and `starting_csv` before or after this part that declares and assigns `disable_security_context_config` and `has_scc_key` variables. Both `install_plan_approval` and `starting_csv` just look out of place here.

Reply:
Interestingly my IDE does not highlight the `local` keyword unless it's at the top of the function (preceded only by other declarations and value assignments). But perhaps this is only a parser's deficiency since shellcheck seems happy 🤞🏻 .